pip-requirements.txt

```diff
@@ -1,8 +0,0 @@
-ckantoolkit==0.0.4
-pika>=1.1.0
-pyOpenSSL==18.0.0
-redis
```

Comment:
so is these already in other places?

Reply:
I believe it's been restructured to no longer need them.